catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/source/TestWfsFilterDelegate.java

```diff
@@ -1,2116 +0,0 @@
-/**
- * Copyright (c) Codice Foundation
- * <p>
- * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
- * General Public License as published by the Free Software Foundation, either version 3 of the
- * License, or any later version.
- * <p>
- * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
- * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
- * is distributed along with this program and can be found at
- * <http://www.gnu.org/licenses/lgpl.html>.
- **/
-package org.codice.ddf.spatial.ogc.wfs.v1_0_0.catalog.source;
-
-import static org.mockito.Mockito.mock;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
-import org.codice.ddf.spatial.ogc.wfs.catalog.common.FeatureMetacardType;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class TestWfsFilterDelegate {
```

Comment:
@kcwire told me this was still an applicable standard and we should keep it. So we should keep the test file and open the ticket to clean up the unit tests.